app-admin/pinky-bar/pinky-bar-9999.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://gitlab.com/void0/pinky-bar.git"
+
+inherit git-r3
+
+DESCRIPTION="Gather some system information and show it in this statusbar program"
+HOMEPAGE="https://gitlab.com/void0/pinky-bar"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="x11 alsa +net libnl +pci dvd sensors ncurses +colours weather mpd drivetemp smartemp"
+
+DEPEND="
+	sys-devel/m4
+	sys-apps/gawk
+	sys-devel/autoconf
+	>=sys-devel/automake-1.14.1
+"
+RDEPEND="
+	alsa? ( media-libs/alsa-lib )
+	x11? ( x11-libs/libX11 )
+	net? ( sys-apps/iproute2 )
+	libnl? ( >=dev-libs/libnl-3.2.27 dev-util/pkgconfig )
+	pci? ( sys-apps/pciutils )
+	dvd? ( dev-libs/libcdio )
+	sensors? ( sys-apps/lm_sensors )
+	ncurses? ( sys-libs/ncurses )
+	weather? ( net-misc/curl app-arch/gzip )
+	mpd? ( media-sound/mpd media-libs/libmpdclient )
+	drivetemp? ( app-admin/hddtemp )
+	smartemp? ( sys-apps/smartmontools )
```

Comment:
a number of these libraries will likely be linked, but given that they're pure `RDEPEND`s, you don't guarantee their presence at build time. Check these again, and add them to a `COMMON_DEPEND` variable instead, which you can include in both `DEPEND` and `RDEPEND`.